google-cloud-tools-plugin/google-cloud-storage/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Use IntelliJ src structure for now
+sourceSets.main.java.srcDirs = ['src']
+// 'src' is included in resources to export .form files to upstream modules.
+sourceSets.main.resources.srcDirs = ['resources', 'src']
+sourceSets.test.java.srcDirs = ['testSrc']
+sourceSets.test.resources.srcDirs = ['testResources']
+
+intellij {
+}
+
+dependencies {
+    compile(project(':google-cloud-tools-plugin:google-cloud-core'))
+
+    compile ('com.google.cloud:google-cloud-storage:1.2.0') {
+        exclude group: 'com.google.http-client', module: 'google-http-client'
+        exclude group: 'com.google.http-client', module: 'google-http-client-jackson'
+        exclude group: 'com.google.code.findbugs', module: 'jsr305'
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'com.google.oauth-client', module: 'google-oauth-client'
+        exclude group: 'com.google.protobuf', module: 'protobuf-java'
+        exclude group: 'com.google.protobuf', module: 'protobuf-java-util'
+    }
+
+    compileOnly 'com.google.auto.value:auto-value:1.4.1'
+    apt         'com.google.auto.value:auto-value:1.4.1'
+
+    testCompile 'xalan:serializer:2.7.1'
```

Comment:
did you find out that these are needed by just running it and observing the errors? Just making sure it wasn't added unnecessarily.

Reply:
good point. I didn't need it. removed